app/Http/Controllers/HomeController.php

```diff
@@ -17,4 +17,16 @@ public function index(TimeLogRepository $timeLogRepository)
         $weekly_summary = $timeLogRepository->weeklySummary();
         return view('welcome', compact('weekly_summary'));
     }
+
+     /**
+     * Show the profile config page.
+     *
+     * @param  App\Modules\Time\Repositories\Interfaces\TimeLogRepositoryInterface $timeLogRepository
+     * @return \Illuminate\Http\Response
+     */
+    public function indexEmployee(TimeLogRepository $timeLogRepository)
```

Comment:
The Employee should have a separate controller. Create a folder named `Admin` and move both admin controllers there (`HomeController` and `ProfileController`). Adjust the namespaces accordingly.
Then, create another folder `Employee` and place the employee's `HomeController` there. 
**Note:** The employee controller doesn't need TimeLogRepository, or any other repository at this point.